app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidget.kt

```diff
@@ -158,7 +173,9 @@ class StaticWidget : AppWidgetProvider() {
                 appWidgetId,
                 entitySelection,
                 attributeSelection,
-                labelSelection
+                labelSelection,
+                textSizeSelection?.toFloatOrNull() ?: 30F,
+                if (separatorSelection == "") " " else separatorSelection ?: " "
```

Comment:
@JBassett I think if user never touches the text field, the default value is an empty string not null, so can I write like this?